samples/nrf_rpc/protocols_serialization/client/README.rst

```diff
@@ -0,0 +1,67 @@
+.. _nrf_rpc_protocols_serialization_client:
+
+Protocols serialization: client
```

Comment:
I'm not sure if I got your ides. nRF RPC is just the library that is used to perform serialization. The main purpose of samples is to have\demonstrate API call serialization of protocols (not serialization itself). To show how to build applications above protocols API with serialization in between.

Reply:
I'm looking at the file naming patterns and tags. This is what I see:

- Sample location: samples/nrf_rpc/protocols_serialization
- Sample tags: nrf_rpc_ps_samples
- Sample names and section name -- no reference to nrf_rpc

And that's why I asked. Given your answer, I don't understand why nRF RPC is mentioned in location and tags if you then say `nRF RPC is just the library that is used to perform serialization.` and don't want to mention it in the sample names.

Reply:
I'm pretty sure such naming will cause misunderstanding since RPC is the protocol based solution on CBOR by itself. However, these samples are for demonstration serialization of Bluetooth, OpenThread etc. RPC is here just the layer above serial transport. 

IMHO it is worth to change sample names.

Nevertheless, if [Bartosz](@bpienk) approves it, I'll change.

@bpienk, please take a look at the topic.

Reply:
I'm fine with whatever name as long as we're consistent in the tags, dir names, and sample titles.

Reply:
Hi @greg-fer, discussed this with @bpienk IRL. Please check changes. Will they satisfy requirements?

Reply:
Yes, thank you for follow-up!